testing/test_tf_serving.py

```diff
@@ -17,6 +17,7 @@
 from __future__ import print_function
 
 import logging
+import time
```

Comment:
So this file is the one that would probably create the junit file.
See what we do in test_runner.py
https://github.com/kubeflow/tf-operator/blob/master/py/test_runner.py#L140

Reply:
I think we can merge this one first if looks good, so we can have the test up and running now.
I will work on adding junit soon.

Reply:
Actually, I think it's small change. I will do it in this PR now.